experiments/AMIP/moist_mpi_earth/coupler_utils/flux_calculator.jl

```diff
@@ -70,15 +70,15 @@ function calculate_surface_fluxes_atmos_grid!(integrator, info_sfc)
     if :ρe_tot in propertynames(Y.c)
 
         flux_energy = ones(axes(dif_flux_energy))
-        parent(flux_energy) .= parent(tsf.shf .+ tsf.lhf .* swap_space!(abs.(ice_mask .- FT(1)), axes(tsf.shf)))  # only SHF above sea ice
+        parent(flux_energy) .= parent(tsf.shf .+ tsf.lhf .* swap_space!(FT(1) .- ice_mask, axes(tsf.shf)))  # only SHF above sea ice
```

Comment:
maybe we could put a comment in about what 1-icemask indicates? I'm forgetting now :)